[test] Random nits on tests (#2019)
taichi-dev/taichi: docs/write_test.rst

        foo()
        assert ti.approx(r[None]) == math.atan2(y, x)

.. note::

>    The order of these decorators doesn't matter.

Reviewer: ```suggestion
   The order of these decorators does not matter.
```

.. code-block:: python

>     # Run this test on all backends except for OpenGL

Reviewer: ```suggestion
    # Run this test on all backends that support sparse computation
```
